view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,69 @@
+ @ipad:  ~"only screen and (min-width: 768px) and (max-width: 1024px)";
+  @mobile:  ~"only screen and (max-width: 767px)";
+.payment-method {
+	.field {
+		label {
+			width: 100%;
+		}
+	}
+	#cardknox_cc_type_cvv_div {
+		.control._with-tooltip {
+			display: inline-block;
+			width: 200px;
+		}
+	}
+}
+
```

Comment:
I removed these few lines and moved down the label.
 label[data-ifields-id="card-data-error"] {
        display: none;
    }

Reply:
@mstein-ck could you reveiw?